### PR TITLE
feat: productize event-broker and fix excessive logging in prod

### LINKED ISSUE #1774: [fxa-event-broker] Finish productizing
A few things that cropped up during deployment we should resolve:
- [x] Create a `__lbheartbeat__` route.
- [x] Implement `process.on` handlers as in `fxa-auth-server` and elsewhere for graceful restarts, logging of unhandled rejections ,etc.
- [x] Make a `simulate_webhook_call` script for use by ops to verify a RP has setup their webhookUrl handler correctly.
- [x] Handle 404's less noisily at debug level.
- [x] Log any malformed messages and ack to avoid clogging the queue.

### PR DESCRIPTION
**Pull request by bbangert**
Closes #1774

### CODE REVIEW

File: packages/fxa-event-broker/README.md [hunk 1]
```diff
@@ -1,3 +1,27 @@
...
+
+    node dist/bin/simulate-webhook-call.js CLIENTID WEBHOOKURL CAPABILITIES
+
+Where `CAPABILITIES` is a comma-seperated string of capabilities to include.
+
+#### Usage
+
+```bash
+
+    $ node run build
```
[the remark this answers is not in the document]
Comment (bbangert) [reply packages/fxa-event-broker/README.md]: That compiles the Typescript to Javascript.

[the remark this answers is not in the document]
Comment (6a68) [reply packages/fxa-event-broker/README.md]: right, `node run build` didn't work for me, but `npm run build` did

File: packages/fxa-event-broker/README.md [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
+    $ export LOG_FORMAT=pretty
+    $ node dist/bin/simulate-webhook-call.js a9238ba https://example.com/webhook capability_1
```
Comment (6a68) [inline packages/fxa-event-broker/README.md]: @bbangert you mentioned you use httpbin for testing. could you paste your working snippet in here?

Comment (bbangert) [reply packages/fxa-event-broker/README.md]: Sure, using say, `https://requestbin.fullcontact.com/1mhpma41?inspect`:

```bash
$ export OPENID_KEYFILE=$REPO_PATH/packages/fxa-auth-server/config/secret-key.json
$ node dist/bin/simulate-webhook-call.js a9238ba https://requestbin.fullcontact.com/1mhpma41 test_capability
```

File: packages/fxa-event-broker/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (bbangert) [reply packages/fxa-event-broker/README.md]: Fixed! Thanks!

**Pull request merged**